examples/plot/package.json

```diff
@@ -5,6 +5,7 @@
     "build": "NODE_ENV=production webpack --env.prod=true"
   },
   "dependencies": {
+    "d3-scale": "^1.0.6",
```

Comment:
This was a missing dependency before the PR was added. Yes it is used to scale the axes.